### PR TITLE
Feat/convert stl ldraw

### PR DESCRIPTION
**Pull request by bgoldbeck**

### CODE REVIEW

File: src/model_conversion/model_shipper.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        # Loop through main model mesh facets
+        ModelShipper._line_type3_to_file(file, model.get_mesh())
+
+        # Loop through child models mesh
+        for i in range(len(model.get_children())):
+            ModelShipper._line_type3_to_file(file, model.get_children()[i])
+
+    @staticmethod
+    def _line_type0_to_file(file, comment):
+        """Write a comment line to the file using LDraw File Format line type 0
```
Comment (sohantamang) [inline src/model_conversion/model_shipper.py]: Looks like the docstring for type0 and type3 methods are switched

File: src/model_conversion/model_shipper.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        :param comment: The comment to write
+        :return: None
+        """
+        file.write("0 // " + comment + "\n")
+
+    @staticmethod
+    def _line_type3_to_file(file, mesh):
+        """Write a line using LDraw File Format line type 3
+        A comment line is formatted:
+        0 // <comment>
```
Comment (sohantamang) [inline src/model_conversion/model_shipper.py]: here too

File: src/model_conversion/model_shipper.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+        :param file_path: The path to the stl file.
+        :return: The BaseStl model (numpy-stl) loaded from the file_path or None.
+        """
+
+        #TODO: confirm valid filepath
+        return Mesh.from_file(file_path)
+
+    @staticmethod
+    def save_ldraw_file_model(file_path, model: LDrawModel):
```
Comment (sohantamang) [inline src/model_conversion/model_shipper.py]: Do we need to close the file `file.close()` at the end by of this method/call another method to do that?

File: src/model_conversion/model_shipper.py (hunk 2 above)
Comment (bgoldbeck) [reply src/model_conversion/model_shipper.py]: Added file close to the main ldraw save function

File: src/model_conversion/model_shipper.py (hunk 1 above)
Comment (bgoldbeck) [reply src/model_conversion/model_shipper.py]: Swapped the comments around.

**Pull request merged**